fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java

```diff
@@ -295,7 +295,9 @@ public HeartbeatResponse call() {
             String url = "http://" + fe.getHost() + ":" + Config.http_port
                     + "/api/bootstrap?cluster_id=" + clusterId + "&token=" + token;
             try {
-                String result = Util.getResultForUrl(url, null, 2000, 2000);
+                String result = Util.getResultForUrl(url, null,
+                                                     FeConstants.heartbeat_interval_second * 1000,
```

Comment:
The constant variable is configurable ?

Reply:
I have changed the title

Reply:
Why FE heartbeat timeout need to same with BE or Broker? Why the heartbeat timeout isn't configurable?